docs/src/tutorials/xla_compilation.md

```diff
@@ -0,0 +1,78 @@
+# Compiling NeuralOperators.jl using Reactant.jl
+
+```@example xla_compilation
+using NeuralOperators, Lux, Random, Enzyme, Reactant
+
+function sumabs2first(model, ps, st, x)
+    z, _ = model(x, ps, st)
+    return sum(abs2, z)
+end
+
+dev = reactant_device()
+```
+
+## Compiling DeepONet
+
+```@example xla_compilation
+deeponet = DeepONet()
+ps, st = Lux.setup(Random.default_rng(), deeponet) |> dev;
+
+u = rand(Float32, 64, 1024) |> dev;
+y = rand(Float32, 1, 128, 1024) |> dev;
+nothing # hide
+
+deeponet_compiled = @compile deeponet((u, y), ps, st)
+deeponet_compiled((u, y), ps, st)[1]
+```
+
+Computing the gradient of the DeepONet model.
+
+```@example xla_compilation
+function ∇deeponet(model, ps, st, (u, y))
+    dps = Enzyme.make_zero(ps)
+    Enzyme.autodiff(
```

Comment:
Minor, but if you'd like you could use dps = Enzyme.gradient(Reverse, ...) here [and mark everything const except dps]